packages/uui-base/lib/mixins/SelectableMixin.ts

```diff
@@ -76,14 +76,20 @@ export const SelectableMixin = <T extends Constructor<LitElement>>(
     constructor(...args: any[]) {
       super(...args);
       this.addEventListener('click', this._handleClick);
-      this.addEventListener('keydown', this.handleSelectKeydown);
+      this.addEventListener('keypress', this.handleSelectKeydown);
```

Comment:
We should use `keypress` event - it has been deprecated for a while know:
https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event

Reply:
Well spotted, thanks. We need to correct this across.

Reply:
And a PR for that: https://github.com/umbraco/Umbraco.UI/pull/960